.tekton/content-sources-frontend-pull-request.yaml

```diff
@@ -241,54 +223,11 @@ spec:
             script: |
               #!/bin/sh
               NODE_VERSION=$(curl -s https://raw.githubusercontent.com/content-services/content-sources-frontend/main/.nvmrc | tr -d '\n' | tr -d '\r')
-              echo "$(tasks.parse-build-deploy-script.results.image)"
               echo "${NODE_VERSION}" > /tekton/results/node-version
         results:
           - name: node-version
             description: The Node.js version from .nvmrc file
 
-    - name: create-frontend-dockerfile
-      taskRef:
-        resolver: git
-        params:
-          - name: url
-            value: https://github.com/RedHatInsights/konflux-consoledot-frontend-build
-          - name: revision
-            value: 2fcdfa9b4858ac941b50ad37317c4f9aaabf91b4
-          - name: pathInRepo
-            value: tasks/create-frontend-dockerfile/create-frontend-dockerfile.yaml
-      workspaces:
-      - name: source
-        workspace: workspace
-      params:
-        - name: path-context
-          value: $(params.path-context)
-        - name: component
-          value: $(tasks.parse-build-deploy-script.results.component)
-        - name: image
-          value: $(tasks.parse-build-deploy-script.results.image)
-        - name: node-build-version
-          value: $(tasks.get-node-version.results.node-version)
-        - name: quay-expire-time
-          value: $(tasks.parse-build-deploy-script.results.quay-expire-time)
-        - name: npm-build-script
-          value: $(tasks.parse-build-deploy-script.results.npm-build-script)
-        - name: yarn-build-script
-          value: build
```

Comment:
Why was this deleted... I just updated this a few days ago! haha

Reply:
- name: yarn-build-script
          value: build
        - name: node-build-version
          value: $(tasks.get-node-version.results.node-version)
          
          do not reference that script

Reply:
[steps](https://docs.google.com/document/d/1bu9bUQcBPC_4yu71WxErw4ty7OPb1JpaRFyHBLPYr_g/edit?tab=t.0) say to remove create-frontend-dockerfile and parse-build-deploy-script tasks 🤷‍♀️  i didn't realize these were just updated until today lol